about/press.html

```diff
@@ -58,7 +120,359 @@
+
+      <p>You can find a copy of the entire press kit <a href="https://www.dropbox.com/sh/ng9p8d5hba425ht/AAB18_F_NZzX_UlQL8LB1caMa?dl=0">here</a>.</p>
+
+      <br>
+      <h3 id="press-releases">Press Releases</h3>
+
+      <p>
+        <i>
+          <b>Rails Girls Summer of Code Kicks-Off its 4th edition with 20 teams from 11 countries </b>
+          <br>
+          (download version in
+          <a href="https://www.dropbox.com/s/xp55d4hg73rlfn2/RGSoC2016Kicks-Off.pdf?dl=0">.pdf</a> or 
+          <a href="https://www.dropbox.com/s/r2whcuni5f4bm3q/RGSoC2016Kicks-Off.txt?dl=0">.txt</a>)
+        </i>
+      </p> 
+
+      <p>
+        <i>
+        <b>Rails Girls Summer of Code raises over 115,000 USD to fund students all over the world </b>
+        (download version in
+          <a href="https://www.dropbox.com/s/civqbyv79tsocyd/RGSoCraisesover100000USDtofundstudentsallovertheworld.pdf?dl=0">.pdf</a> or 
+          <a href="https://www.dropbox.com/s/2mvnhiyc8gomdea/RGSoCraisesover100000USDtofundstudentsallovertheworld.txt?dl=0">.txt</a>)
+        </i>
+      </p>
+
+      <p>
+        <i>
+          <b>Rails Girls Summer of Code: Ruby Hero Award winners </b>
+          <br>
+          (download version in
+            <a href="https://www.dropbox.com/s/rmlta81q9cyo376/RGSoC-%20The%20first%20%E2%80%9CSummer%20of%20Code%E2%80%9D%20for%20women%20is%20launched.pdf?dl=0">.pdf</a> or 
+            <a href="https://www.dropbox.com/s/vfw1iiejydnxhem/RGSoC-%20The%20first%20%E2%80%9CSummer%20of%20Code%E2%80%9D%20for%20women%20is%20launched.txt?dl=0">.txt</a>)
+        </i>
+      </p> 
+
+      <p>
+        <i>
+          <b>Rails Girls Summer of Code: survey shows 90% of their students are still working in tech </b>
+          (download version in
+            <a href="https://www.dropbox.com/s/rmlta81q9cyo376/RGSoC-%20The%20first%20%E2%80%9CSummer%20of%20Code%E2%80%9D%20for%20women%20is%20launched.pdf?dl=0">.pdf</a> or 
+            <a href="https://www.dropbox.com/s/vfw1iiejydnxhem/RGSoC-%20The%20first%20%E2%80%9CSummer%20of%20Code%E2%80%9D%20for%20women%20is%20launched.txt?dl=0">.txt</a>)
+        </i>      
+      </p>
+
+      <p>
+        <i>
+          <b>Rails Girls Summer of Code: The first “Summer of Code” for women is launched </b>
+          (download version in
+            <a href="https://www.dropbox.com/s/rmlta81q9cyo376/RGSoC-%20The%20first%20%E2%80%9CSummer%20of%20Code%E2%80%9D%20for%20women%20is%20launched.pdf?dl=0">.pdf</a> or 
+            <a href="https://www.dropbox.com/s/vfw1iiejydnxhem/RGSoC-%20The%20first%20%E2%80%9CSummer%20of%20Code%E2%80%9D%20for%20women%20is%20launched.txt?dl=0">.txt</a>)
+        </i>
+      </p>
+
+      <br>
+      <h3 id="logos">Logos</h3>
 
       <p>Coming soon. </p>
-      
\ No newline at end of file
+
+      <br>
+      <h3 id="images">Images</h3>
+
+      <p>Below are samples from teams of the Rails Girls Summer of Code during 2015 and 2016. There are many more in our Press Kit.</p>
```

Comment:
As suggested by @anasofiapinho, since we have other types of images in dropbox, we might want to change the wording slightly to something like: 
Below are samples from teams of the Rails Girls Summer of Code during 2015 and 2016. There are many more, as well as other images, in our Press Kit.